persistence-cassandra/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/cassandra/CassandraConfigProvider.scala

```diff
@@ -16,11 +16,10 @@ import scala.collection.immutable
  * Internal API
  */
 // Injecting ActorSystem and not Configuration because Configuration isn't always bound when running tests
-@Singleton
-final class CassandraConfigProvider @Inject() (system: ActorSystem) extends Provider[CassandraConfig] {
+final class CassandraConfigProvider(system: ActorSystem) {
```

Comment:
do we need this class at all? move the code to `CassandraConfigImpl.apply(system: ActorSystem): CassandraConfigImpl` ?